ibm_catalog.json

```diff
@@ -1007,6 +1007,256 @@
           ],
           "dependency_version_2": true,
           "terraform_version": "1.10.5"
+        },
+        {
+          "label": "Quickstart",
+          "name": "quickstart",
+          "index": 1,
+          "install_type": "fullstack",
+          "working_directory": "solutions/quickstart",
+          "compliance": {
+            "authority": "scc-v3",
+            "profiles": [
+              {
+                "profile_name": "IBM Cloud Framework for Financial Services",
+                "profile_version": "1.7.0"
+              }
+            ]
+          },
+          "iam_permissions": [
+            {
+              "service_name": "containers-kubernetes",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "notes": "Required to reset API keys, create and edit the OpenShift cluster, and manage all related resources."
```

Comment:
Required to reset API keys --> Required to reset the cluster API key

(try be consistent with wording in official [docs](https://cloud.ibm.com/docs/containers?topic=containers-access-creds#admin-set-credentials))